Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts as build-stage
+
+# Install deps
+RUN apt-get update -y \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -y wget=1.* \
+    && rm -rf /var/lib/apt/lists/*
+RUN wget --progress=dot:giga -O nvim.appimage \
+  'https://github.com/neovim/neovim/releases/download/v0.4.4/nvim.appimage'
```

Comment:
I think it'd be better if this used either the nightly or stable tags. I don't want to have to remember to update this file every time a new version comes out :)

Reply:
Will do and ping you once done

Reply:
Done